__tests__/deleteUsers/README.md

```diff
@@ -0,0 +1,52 @@
+# DeleteUsers Tests
+
+All the tests data for deleteUsers are to be present in __tests__/data/deleteUsers/${destination}/
+
+### Files and their significance
+  - __`handler_input.json`__ - Input data for  `handleDeletionOfUsers` function in `versionedRouter.js`(alias for `_deleteUsers_proxy_input.json`)
+  - __`handler_output.json`__ - Output of `handleDeletionOfUsers` function in `versionedRouter.js`(alias for `_deleteUsers_proxy_output.json`)
+  - __`prepare_req_input.json`__ - Input data for `prepareDeleteRequest` function exported from the destination's `deleteUsers.js` (An example can be seen in the case of `ga`)
```

Comment:
Delete prepare*.json files & add note on `reqParams` in `nw_client_data.json` getting validated

Reply:
I have made changes, can you verify ?